learn/what_is_meilisearch/features.md

```diff
@@ -54,6 +54,10 @@ If you enclose search terms in double quotes (`"`), MeiliSearch will only return
 
 Phrase search is particularly useful when looking for a single result, such as searching for a book by ISBN. It is also possible to combine phrase searches with MeiliSearch's basic syntax so only parts of a query are matched strictly.
 
+## API key management
+
+MeiliSearch allows you to protect your instances through the use of API keys. API keys allow you to control which users can access what indexes, routes, and endpoints. These API keys are managed by the master key. When you set a master key on your first launch, MeiliSearch generates two API keys called the `Default Search API Key` and the `Default Admin API Key`. You can read more them in our article on [security](/reference/features/security.md).
```

Comment:
A couple minor edits:

1. Avoiding starting two sentences with "[thing] allows you to [do something]";
2. Breaking the paragraph in two for a bit easier reading? (Might be unnecessary, feel free to ignore it if you disagree). 

```suggestion
MeiliSearch allows you to protect your instances through the use of API keys. API keys give you control over which users can access what indexes, routes, and endpoints.

API keys are managed by the master key. When you set a master key on your first launch, MeiliSearch generates two API keys called the `Default Search API Key` and the `Default Admin API Key`. You can read more them in our article on [security](/reference/features/security.md).
```